def raise_exception():
    raise TypeError("Exception raised")


raise_exception()